scripts/pre_commit_linter.py

```diff
@@ -340,21 +362,49 @@ class FileCache(object):
 
     @classmethod
     def read(cls, filepath, mode='r'):
-        """Returns the data read from the file.
+        """Returns the data read from the file in unicode form.
 
         Args:
             filepath: str. The file path from which data is to be read.
             mode: str. The mode in which the file is to be opened.
 
         Returns:
-            str. The data read from the file.
+            unicode. The data read from the file.
         """
         return cls._get_data(filepath, mode)[0]
 
+    @classmethod
+    def read_as_str(cls, filepath, mode='r'):
```

Comment:
We should always be reading as unicode and using unicode internally.

Reply:
@brianrodri suggested that we put the encoding/decoding part in the `FileCache` class [here](https://github.com/oppia/oppia/pull/6613#discussion_r277507776). Should I revert the changes?

Reply:
There shouldn't even be such a function -- all our read() functions should just output unicode, so that the rest of the codebase doesn't have to deal with any ambiguity.

Reply:
Done